lib/Sema/DerivedConformanceEquatableHashable.cpp

```diff
@@ -665,60 +666,43 @@ deriveEquatable_eq(TypeChecker &tc, Decl *parentDecl, NominalTypeDecl *typeDecl,
                                     FunctionType::ExtInfo());
   }
   eqDecl->setInterfaceType(interfaceTy);
-  eqDecl->copyFormalAccessFrom(typeDecl, /*sourceIsParentContext*/true);
+  eqDecl->copyFormalAccessFrom(derived.Nominal, /*sourceIsParentContext*/ true);
```

Comment:
should be `derived.ConformancDecl` here?

Reply:
I don't think it makes sense to `copyFormalAccessFrom` from an `extension`?